File: src/main/java/derg/explodee/Explodee.java
package derg.explodee;

import net.fabricmc.api.ModInitializer;
import net.minecraft.core.net.command.CommandManager;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import turniplabs.halplibe.util.GameStartEntrypoint;

public class Explodee implements ModInitializer, GameStartEntrypoint {
	public static final String MOD_ID = "explodee";
	public static final Logger LOGGER = LoggerFactory.getLogger(MOD_ID);

	@Override
	public void onInitialize() {}

	@Override
	public void beforeGameStart() {
		CommandManager.registerCommand(new SummonExplosionCommand());
	}

	@Override
	public void afterGameStart() {}
}

File: src/main/kotlin/derg/explodee/Explodee.kt
package derg.explodee

import net.fabricmc.api.ModInitializer
import org.slf4j.Logger
import org.slf4j.LoggerFactory
import turniplabs.halplibe.util.GameStartEntrypoint

object Explodee : ModInitializer, GameStartEntrypoint {
    const val MOD_ID: String = "explodee"
    val LOGGER: Logger = LoggerFactory.getLogger(MOD_ID)

    override fun onInitialize() {}

    override fun beforeGameStart() {}

    override fun afterGameStart() {}
}
